cmd/api-server/main.go

```diff
@@ -242,6 +243,17 @@ func main() {
 				log.DefaultLogger.Infow("minio is not available, using default logs storage", "error", err)
 			}
 		}
+
+		// Run DB migrations
+		dbMigrator, err := dbmigrator.NewDbMigrator(db, "__migrations", cfg.TestkubeConfigDir)
```

Comment:
might be it will be good to enable/disable migrations in helm chart

Reply:
Added `DISABLE_MONGO_MIGRATIONS` environment variable, and configured it via `disableMongoMigrations` in Helm chart (https://github.com/kubeshop/helm-charts/pull/662)

Reply:
cool